ee/clickhouse/views/test/__snapshots__/test_clickhouse_stickiness.ambr

```diff
@@ -22,7 +22,7 @@
      GROUP BY aggregation_target)
   WHERE num_intervals <= 8
   GROUP BY num_intervals
-  ORDER BY num_intervals SETTINGS optimize_move_to_prewhere = 0
```

Comment:
Changes to the snapshot are because the move to settings on `client.execute` don't show here?

Reply:
Yep - they're not part of the query string that's sent out. I think we shouldn't be setting settings in-sql anyways :D